app/forms/collection_form.rb

```diff
@@ -3,91 +3,71 @@
-
-  def persisted?
-    druid.present?
-  end
-
+  attribute :email_depositors_status_changed, :boolean, default: true
+  attribute :email_when_participants_changed, :boolean, default: true
+  attribute :license, :string
+  attribute :license_option, :string, default: 'required'
   attribute :lock, :string
-
-  attribute :version, :integer, default: 1
-
+  attribute :provided_custom_rights_statement, :string
+  attribute :release_duration, :string
+  attribute :release_option, :string, default: 'immediate'
+  attribute :review_enabled, :boolean, default: false
   attribute :title, :string
-  validates :title, presence: true
+  attribute :version, :integer, default: 1
+  attribute :work_subtypes, array: true, default: -> { [] }
+  attribute :work_type, :string
 
-  # The Collection description maps to the cocina abstract
-  attribute :description, :string
-  validates :description, presence: true
+  alias id druid
 
-  attribute :access, :string, default: 'world'
+  # rubocop:disable Rails/I18nLocaleTexts
   validates :access, inclusion: { in: %w[world stanford depositor_selects] }
-
-  attribute :license_option, :string, default: 'required'
-  validates :license_option, inclusion: { in: %w[required depositor_selects] }
-
-  attribute :license, :string
-  validates :license, presence: true, if: -> { license_option == 'required' }
-  attribute :default_license, :string
-  validates :default_license, presence: true, if: -> { license_option == 'depositor_selects' }
-
-  attribute :custom_rights_statement_option, :string, default: 'no'
+  validates :custom_rights_statement_instructions, presence: true,
+                                                   if: -> { custom_rights_statement_option == 'depositor_selects' }
   validates :custom_rights_statement_option, presence: true, inclusion: { in: %w[no provided depositor_selects] }
-
-  attribute :provided_custom_rights_statement, :string
+  validates :default_license, presence: true, if: -> { license_option == 'depositor_selects' }
+  validates :deposits_contact_email, format: {
+    with: URI::MailTo::EMAIL_REGEXP, allow_blank: true, message: I18n.t('contact_email.validation.email.invalid')
+  }
+  validates :description, presence: true
+  validates :doi_option, inclusion: { in: %w[yes no depositor_selects] }
+  validates :license, presence: true, if: -> { license_option == 'required' }
+  validates :license_option, inclusion: { in: %w[required depositor_selects] }
+  validates :managers_attributes, length: { minimum: 1, message: 'must have at least one manager' }
   validates :provided_custom_rights_statement, presence: true, if: -> { custom_rights_statement_option == 'provided' }
-  before_validation do
-    self.provided_custom_rights_statement = LinebreakSupport.normalize(provided_custom_rights_statement)
-  end
+  validates :release_duration, presence: { message: 'select a valid duration for release' },
+                               if: -> { release_option == 'depositor_selects' }
```

Comment:
This used to be a `validate` call pointing at a method, but I didn't see a reason to deviate from the pattern we use for all the other validations.